addons/debugger/performance.js

```diff
@@ -1,10 +1,9 @@
+import "../../libraries/thirdparty/cs/chart.min.js";
```

Comment:
It may be better for this to be a dynamic import(). A lot of the time, this module is imported but never used, so statically importing chart.js (which takes a non-trivial amount of time) is unnecessary.